internal/proxyservice/proxy_service.go

```diff
@@ -391,17 +342,10 @@ func (s *proxyServices) credsRetriever(
 	creds []*v2.Credential,
 ) internal.CredentialsRetriever {
 	return func() (map[string][]byte, error) {
-		return s.getSecrets(creds)
+		return s.resolver.Resolve(creds)
 	}
 }
 
-// getSecrets returns the secret values for the requested credentials.
-// TODO: Move this up one level, pass it down as dep.  Danger: This has
-//   a hardcoded dependency on plugin and v1.
-func (s *proxyServices) getSecrets(secrets []*v2.Credential) (map[string][]byte, error) {
-	return s.resolver.Resolve(secrets)
-}
-
```

Comment:
@sgnn7 Any idea how this change go in here?  This change was made a while ago iirc and I didn't do this (I don't think) as part of this PR.